src/frontend/scss/components/templates/richie/large_banner/_large_banner.scss

```diff
@@ -223,3 +223,153 @@
     }
   }
 }
+
+// A compact banner without image contents
+// Opposed to 'hero-intro' this variant delegates the top padding for topbar in 'over'
+// mode in content component
+//
+.compacted-banner {
+  position: relative;
+  padding: 0;
```

Comment:
What do you think to move this add into a dedicated file ? `large_banner/_compacter_banner.scss` ?

Reply:
Done